e2e-tests/tfm/cypress/e2e/pages/caseDealPage.js

```diff
@@ -28,6 +28,16 @@ const caseDealPage = {
       };
     },
   },
+
+  eligibilityCriteriaTable: {
+    row: (index) => {
+      cy.get(`[data-cy="criterion-${index}-row"]`).as('row');
+      return {
+        answerTag: () => cy.get('@row').get('[data-cy="eligibility-criteria-answer-tag"'),
+        heading: (criterionId) => cy.get('@row').get(`[data-cy="criterion-${criterionId}-heading"]`),
+      };
+    },
+  },
```

Comment:
I know this is how the `dealFacilitiesTable` object above works, but this is a bit fragile because we're overwriting the Cypress alias `row` each time we call `row(<some value>)`. E.g. if we wrote

```
const row1 = eligibilityCriteriaTable.row(1);

// This checks the answer tag exists for the first row, great.
row1.answerTag().should('exist'); 

const row2 = eligibilityCriteriaTable.row(2);

// This checks the second row, all good.
row2.answerTag().should('exist');

// This checks the _second_ row because creating `row2` redefined
// the `row` alias to point to the 2nd row.
row1.answerTag().should('exist');
```

We'll have a similar issue if we start mixing `eligibilityCriteriaTable.row(...)` and `dealFacilitiesTable.row(...)`

Can we use more specific aliases to fix this please?